bigbluebutton-html5/imports/api/2.0/audio/client/bridge/verto.js

```diff
@@ -17,8 +17,8 @@ export default class VertoBridge extends BaseAudioBridge {
     window.vertoExitAudio();
   }
 
-  joinListenOnly() {
-    window.vertoJoinListenOnly(
```

Comment:
why did you change the name from joinListenOnly to joinAudio? And change them both to join microphone?